game_data.py

```diff
@@ -1,12 +1,41 @@
-# structure defining the rooms in the game. Try adding more rooms to the game!
-rooms = {
-    "Tavern": {
-        "description": "You're in a cozy tavern warmed by an open fire.",
-        "exits": {"outside": "Outside"},
-    },
-    "Outside": {
-        "description": "You're standing outside a tavern. It's raining.",
-        "exits": {"inside": "Tavern"},
-    }
-}
+from lib.models.entity import Room
+from lib.models.enums import ExitType
 
+rooms = {}
+
+def add_room(room: Room):
```

Comment:
What's your thinking with this function? It's no different that manually calling the dict

Reply:
I think it's just a wee bit nicer than

`rooms['tavern'] = Room(name = 'tavern'...`

Less chance of human error.